package-lock.json

```diff
@@ -8661,7 +8667,7 @@
     "ansicolors": {
       "version": "0.3.2",
       "resolved": "https://registry.npmjs.org/ansicolors/-/ansicolors-0.3.2.tgz",
-      "integrity": "sha512-QXu7BPrP29VllRxH8GwB7x5iX5qWKAAMLqKQGWTeLWVlNHNOpVMJ91dsxQAIWXpjuW5wqvxu3Jd/nRjrJ+0pqg==",
+      "integrity": "sha1-ZlWX3oap/+Oqm/vmyuXG6kJrSXk=",
```

Comment:
is that change expected here? You are only updating the TypeScript types in this PR.

Reply:
Well, that comes from installation of `@types/crypto-js` package

Reply:
but that should be done by `crypto-js` package, and this package contains just TypeScript types for it